Build warning on idfv5.0.

`    task_val = xTaskCreatePinnedToCore(
        (TaskFunction_t)        audio_task,
                                "Audio Task",
                                4 * 1024,
                                &instance,
        (UBaseType_t)           instance.config.priority,
        (TaskHandle_t * const)  NULL,
                                0);
`

```
../managed_components/chmorgan__esp-audio-player/audio_player.cpp:568:9: warning: type qualifiers ignored on cast result type [-Wignored-qualifiers]
  568 |         (TaskHandle_t * const)  NULL,
      |         ^
```

--- a/audio_player.cpp
+++ b/audio_player.cpp
@@ -565,7 +565,7 @@ esp_err_t audio_player_new(audio_player_config_t config)
                                 4 * 1024,
                                 &instance,
         (UBaseType_t)           instance.config.priority,
-        (TaskHandle_t * const)  NULL,
+        (TaskHandle_t *)  NULL,
         (BaseType_t)            instance.config.coreID);
 
     ESP_GOTO_ON_FALSE(pdPASS == task_val, ESP_ERR_NO_MEM, cleanup,

